lib/Dialect/AIEVec/Transforms/VectorToVectorConversions.cpp

```diff
@@ -11,6 +11,7 @@
 // it compatible with the available vector instructions in AIE architectures
 //===----------------------------------------------------------------------===//
 
+#include "aie/Dialect/AIE/IR/AIEDialect.h"
```

Comment:
For my own understanding, why do we need to include this header?

Reply:
We don't, I added it for one of my first attempts at this, and I forgot to take it out once it wasn't necessary. Good catch!